crates/polars-plan/src/dsl/function_expr/peaks.rs

```diff
@@ -1,59 +1,36 @@
+use polars_core::with_match_physical_numeric_polars_type;
 use polars_ops::chunked_array::peaks::{peak_max as pmax, peak_min as pmin};
 
 use super::*;
 
 pub(super) fn peak_min(s: &Series) -> PolarsResult<Series> {
+    let s = s.to_physical_repr();
```

Comment:
@orlp I went with physical, so we have less monomorphization.

Reply:
I believe that's fine in this case since we only care about order.